src/components/files/fileHandler.ts

```diff
@@ -0,0 +1,119 @@
+import path from 'path'
+import fs from 'fs'
+import util from 'util'
+import { pipeline } from 'stream'
+import { nanoid } from 'nanoid'
+import { getAppRootDir } from '../utilityFunctions'
+import * as config from '../../config.json'
+import DBConnect from '../databaseConnect'
+import createThumbnail from './createThumbnails'
+const sharp = require('sharp')
+
+export const { filesFolderName } = config
+export const filesPath = path.join(getAppRootDir(), filesFolderName)
+
+interface HttpQueryParameters {
+  [key: string]: string
+}
+
+export function createFilesFolder() {
+  try {
+    fs.mkdirSync(path.join(getAppRootDir(), filesFolderName))
+  } catch {
+    // Folder already exists
+  }
+  // Move generic thumbnails to files root
+  fs.readdir(path.join(getAppRootDir(), 'images', 'generic_file_thumbnails'), (_, files) => {
```

Comment:
we can just create that folder on server (can add git ignore inside those folders, to git ignore anything that's added and to make sure folders structure is committed)

Reply:
Yeah, but I've made the "root files folder" a configurable value (in .config). It's currently set to "files", but it could be changed, which is why I didn't include "files" in the repo.

Reply:
sweet